run.py

```diff
@@ -174,6 +174,17 @@ def main(args):
   from nab.detectors.skyline.skyline_detector import SkylineDetector
   from nab.detectors.random.random_detector import RandomDetector
   from nab.detectors.null.null_detector import NullDetector
+  detectors = []
+  for detector in args.detectors:
+      if ':' in detector:
```

Comment:
Please add a comment below this if statement to specify the intention -- i.e. import detectors from other packages.

Also, our standard is to use 2-space indents.